app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -60,6 +61,8 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 }
             });
             bottomNavigation.setSelectedItemId(getIntent().getIntExtra(RES_ITEM, R.id.dashboard));
+            if (isTestRunning())
+                return;
```

Comment:
Please add `{` and `}`: too many errors caused by not adding them.

Please explain why we stop at that point.

Please explain whether this applies to all tests or only tests using fastlane.

Reply:
apply to every scenario when you run UI tests

Reply:
@lorenzoPrimi ^